transmart-rest-api/src/groovy/org/transmartproject/rest/marshallers/QueryResultSerializationHelper.groovy

```diff
@@ -0,0 +1,37 @@
+package org.transmartproject.rest.marshallers
+
+import grails.rest.Link
+import org.transmartproject.core.querytool.QueryResult
+
+import static grails.rest.render.util.AbstractLinkingRenderer.RELATIONSHIP_SELF
+import static org.transmartproject.rest.marshallers.MarshallerSupport.getPropertySubsetForSuperType
+
+/**
+ * Serialization of {@link QueryResult} objects.
+ */
+class QueryResultSerializationHelper extends AbstractHalOrJsonSerializationHelper<QueryResult> {
```

Comment:
One already exists in https://github.com/thehyve/transmart-upgrade/blob/master/transmart-rest-api/src/main/groovy/org/transmartproject/rest/marshallers/QueryResultSerializationHelper.groovy.
Please use that one and make sure that the source files are put under `src/main/groovy` instead of `src/groovy`.